utils/rpms/packaging/Dockerfile.ubuntu.20.04

```diff
@@ -17,10 +17,12 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
             autoconf bash ca-certificates curl debhelper dh-make        \
             dpkg-dev dh-python doxygen gcc git git-buildpackage locales \
             make patch pbuilder pkg-config python3-dev python3-distro   \
-            python3-distutils rpm scons wget
+            python3-distutils rpm scons wget cmake valgrind
```

Comment:
Why does this PR need to change the packaging list?

Reply:
Just to be up-to-date with the packaging [pseudo-]submodule: https://github.com/daos-stack/packaging/blob/456c868721e4518215e01c0beeb6d93ef54d0d19/Dockerfile.ubuntu.20.04#L20